docs/reference/3-admin-api/whats-new.md

```diff
@@ -0,0 +1,166 @@
+# What's New
+
+This section provides an overview of what's new in the latest versions of the Admin API.
+
+**Contents**
+
+* [Updates in Admin API v2.2 (Latest Release)](#updates-in-admin-api-v22-latest-release)
+* [Updates in Admin API v2.1](#updates-in-admin-api-v21)
+* [Updates in Admin API v2.0](#updates-in-admin-api-v20)
+* [Updates in Admin API v1.4](#updates-in-admin-api-v14)
+* [Updates in Admin API v1.3](#updates-in-admin-api-v13)
+* [Updates in Admin API v1.2](#updates-in-admin-api-v12)
+* [Updates in Admin API v1.1](#updates-in-admin-api-v11)
+* [Updates in Admin API v1.0](#updates-in-admin-api-v10)
+
+# Updates in Admin API v2.2 (Latest Release)
+
+Please see the [direct source code updates to Admin API v2.2 on GitHub here](https://github.com/Ed-Fi-Alliance-OSS/AdminAPI-2.x/releases/tag/v2.2.0).
+
+**.NET 8 Upgrade**
+
+Admin API 2.2 includes a technology upgrade from .NET 6 to .NET 8 for long-term support.    [ADMINAPI-983](https://tracker.ed-fi.org/browse/ADMINAPI-983)
```

Comment:
Best to remove all of these links to Tracker. [here is](https://github.com/Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io/blob/main/odsApi_versioned_docs/version-7.2/whats-new/release-notes.md) how I handled Jira ticket numbers in the ODS/API release notes.